### ISSUE TITLE
Twitter widget doesn't display the tweets/timelines

### ISSUE THREAD

**Issue by pandurx** [gctools-outilsgc/gcconnex#90]
Twitter API has changed

**Comment by KevinChagnon** [comment]
As a user, I would like for the Twitter widget to display feeds made to and from a specific twitter profile using Twitter's "list" widget.

The widget works to display current feeds here: https://gcconnex.gc.ca/groups/profile/17575821/shared-travel-services-services-de-voyage-partages

from @Travel.GoC and @Voyage.Gdc 

The widget has been active on this group for a long time.

When looking at the source code used for these widgets, the window is blank, no source code can be seen.

If a new widget is created for the same Twitter feeds, the widget will not work.

User tried to create a new widget on the following page and it does not work:

https://gcconnex.gc.ca/groups/profile/25915180/real-property-services

User is creating a "list" widget on Twitter and using that source code to put in GCConnex's Twitter search widget.

I am not sure if we did have a "list" widget in the past and a "search" widget or if our (search) widget used to work with any type of Twitter widget source code.

In summary; list widgets created in the past still work, new widgets will not work.

Can we find out if we can make this operational once again?

**Comment by LemieuxGen** [comment]
I did a bit more digging, and old list created in the past works in the GCconnex Widget for Twitter Search. The new widgets created in Twitter no longer work in GCconnex (aside from the search widget). Please updated the GCconnex widget to access new embedded code from Twitter. 
Of possible interest: [GCpedia instructions on how to create a Twitter Widget](http://www.gcpedia.gc.ca/wiki/GCconnex_User_Help/Content_Management_and_Collaboration/How_Do_I_Configure_the_Twitter_Widget%3F): 
A Twitter need to be created first in Twitter (in settings - https://twitter.com/settings/widgets)
FYI - @KevinChagnon @gabriellehubert @pandurx

**Comment by ToferC** [comment]
Also looked into. You can create a Twitter search using a #, but otherwise the new twitter embeds don't work.  Approved for work.

**Comment by nschonni** [comment]
This works again in the latest version of WET https://github.com/wet-boew/wet-boew/pull/7885 so you could just leverage that

**Comment by pandurx** [comment]
What is not working? (not familiar with Twitter)

**Comment by LemieuxGen** [comment]
@pandurx , I tested the Twitter Widget on Preprod. A few comments:
- Widget works well, didn't find any major bugs.
- When adding a custom title, the title of the widget doesn't change until you refresh the page. (no sure if anything can be done for that, but it makes it look like a bug at first).
- Now that the GCconnex Twitter widget works for all types of Twitter widget, including the Twitter Search, we shouldn't need to have a second GCconnex widget for "Twitter Search".  I suggest removing the "Twitter search" widget. 
- Is there a reason why we can only add 1 Twitter widget on GCconnex? There could be value in adding more then 1 Twitter widget. I suggest removing this restriction, if possible.

Note: I will send you the translation via email. Thanks!
